setup_chef_cookbooks.sh

```diff
@@ -51,13 +51,13 @@ ENV['no_proxy'] = no_proxy_string
 http_proxy ENV['http_proxy']
 https_proxy ENV['https_proxy']
 no_proxy no_proxy_string
-
+ENV['GIT_SSL_NO_VERIFY'] = 'true'
```

Comment:
Oh yes; I missed that was in the embedded `knife.rb`; someday this will not be terrible shell scripts...